ansible/library/conn_graph_facts.py

```diff
@@ -399,6 +399,8 @@ def build_results(self, hostnames, ignore_error=False):
             if device is None and not ignore_error:
                 msg = "Cannot find device {}, check if it is in {}".format(hostname, self.csv_files["devices"])
                 return (False, msg)
+            if device is None:
+                continue
```

Comment:
Will this `continue` cause any unexpected error in other ansible or testcase?

Reply:
When device is None here, there is a device["Type"] a few lines after which will certainly cause exception and stop execution, which I believe is against the concept of ignore_error.